nexus/src/app/background/blueprint_load.rs

```diff
@@ -153,10 +153,34 @@ impl BackgroundTask for TargetBlueprintLoader {
                                 "status": "target blueprint unchanged (error)",
                                 "error": message
                             })
+                        } else if old.0.enabled != new_bp_target.enabled {
+                            // The blueprints have the same contents, but its
+                            // enabled bit has flipped.
+                            let status = if new_bp_target.enabled {
+                                "enabled"
+                            } else {
+                                "disabled"
+                            };
+                            info!(
+                                log,
+                                "target blueprint enabled state changed";
+                                "target_id" => &target_id,
+                                "time_created" => &time_created,
+                                "state" => status,
+                            );
+                            self.last =
+                                Some(Arc::new((new_bp_target, new_blueprint)));
+                            self.tx.send_replace(self.last.clone());
+                            json!({
+                                "target_id": target_id,
+                                "time_created": time_created,
+                                "time_found": chrono::Utc::now().to_string(),
```

Comment:
Is the `to_string` really necessary?

Reply:
Doesn't look like it is, no. This was copy/pasted from nearby; I'll remove the unnecessary `to_string()`s.